Assignment/solution_student/css/style.css

```diff
@@ -31,19 +31,252 @@ html {
 /* BASE STYLES
 /*******************************************/
 body {
-  color: black;
+  color: #121212;
   font-size: 16px;
-  background: white;
+  background: #ffffff;
+  margin: 0 auto;
+  font-family: 'Open Sans', sans-serif;
 }
-
 /******************************************
 /* LAYOUT
 /*******************************************/
+nav {
+  float: right;
+  margin-right: 50px;
+  margin-top: 20px;
+}
+
+nav li {
+  margin-right: 20px;
+  display: inline;
+}
+
+header {
+  width: 100%;
+  background-color: #033048;
+}
+
+.logo {
+  margin-top: 0;
+  dislay: inline-block;
+}
+
 
 /******************************************
-/* PAGES
+/* SECTIONS
+/*******************************************/
+
+nav a {
+  text-decoration: none;
+  color: #f9e42e;
+  font-style: 700;
+}
+
+.logo {
+  font-size: 24px;
+  font-weight: 800;
+}
+
+header .logo {
+  color: #f0efef;
+  display: inline-block;
+  margin-left: 50px;
+  margin-top: 10px;
+
+}
+
+.content-wrapper{  
+  background: #f0efef;
+  margin: 0 auto;
+  width: 75%;
+  padding: 25px 50px 20px;
+
+}
+
+.articles,
+.sidebar {
+  float: left;
+}
+
+.articles {
+  width: 75%;
+  padding-left: 50px;
+}
+
+.article_images {
+  width: 100%;
+}
+
+.sidebar {
+  width: 25%;
+  background: #f0efef;
+  padding-left: 20px;
+}
+
+.160x600 {
+  padding-left: 20px;
+  padding-top: 60px;
+  padding-right: 50px;
+  float: left;
+}
+
+ul {
+  margin: 0;
+  list-style-type: none;
+}
+
+
+/******************************************
+/* PAGES --- I've actually come to the conclusion
+that i have no idea what "pages" mean or what to potentially
+put here
 /*******************************************/
 
 /******************************************
 /* SECTIONS
-/*******************************************/
\ No newline at end of file
+/*******************************************/
+h2 {
+  color: #033048;
+  font-size: 40px;
+  font-weight: 600;
+  margin-bottom: 20px;
+}
+
+h3 {
+  margin: 0;
+  font-size: 25px;
+  /*32 per the design file looked crazy big*/
+  margin-top: 40px;
+  font-weight: 900;
+  color: #033048;
+}
+
+p {
+  color: #606161;
+  font-size: 24px;
+  line-height: 1.5;
+}
+
+.sidebar p {
+  font-size: 20px;
+  font-weight: 200;
+  color: #606161;
+  margin: 0;
+}
+
+.sidebar a {
+  font-size: 22px;
+  font-weight: 700;
+  text-decoration: none;
+  color: #033048;
+}
+
+article a {
+  color: #033048;
+  font-weight: 700;
+  font-size: 24px;
+  text-align: right;
+  /*I've text aligned right, but to 
+  no avail... What am I doing wrong here?*/
+  text-decoration: none;
+}
+
+.first_character {
+  float: left;
+  padding-right: 10px;
+  font-size: 70px;
+  line-height: 60px;
+  color: #033048;
+
+}
+
+.sidebar li {
+  color: #606161;
+  font-size: 20px;
+  text-decoration: none;
+  margin: 0;
+  /*I have this weird indent... why?*/
```

Comment:
The indent you are seeing is actually `padding` on the `<ul>` — `ul`s by default have some left padding applied to them (http://d.pr/i/1j2Tb), so if you don't want that indent, you'd have to do something like `.sidebar ul {padding: 0;}`